recipes/liburing/build.sh

```diff
@@ -0,0 +1,4 @@
+./configure --cc="$GCC" --cxx="$GXX" --prefix="$PREFIX"
```

Comment:
Can you try whether `--disable-static` works? That would be a better alternative to creating and then instantly deleting the static archives.

Reply:
I tried in https://github.com/conda-forge/staged-recipes/pull/25597/commits/1d6411ca2620ae0c948fe861892eeaedd14983ce, but there is no such option.